### PR TITLE
enable k8s proxy with dns on windows

### PR DESCRIPTION
**Pull request by anhowe**

### CODE REVIEW

File: docs/kubernetes-build-win-binaries.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+6. Build windows kubelet.exe
+ 1. `cd $HOME`
+ 2. `git clone https://github.com/kubernetes/kubernetes $GOPATH/src/k8s.io/kubernetes`
+ 3. `cd $GOPATH/src/k8s.io/kubernetes`
+ 4. `make WHAT=cmd/kubelet`
+ 5. `make WHAT=cmd/kubelet KUBE_BUILD_PLATFORMS=windows/amd64
`
+
+7. Build windows kube-proxy.exe
+ 1. cd $GOPATH/src/k8s.io/kubernetes
```
Comment (JiangtianLi) [inline docs/kubernetes-build-win-binaries.md]: Add single quote for code region?

File: docs/kubernetes.windows.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
+# Microsoft Azure Container Service Engine - Kubernetes Walkthrough
```
Comment (JiangtianLi) [inline docs/kubernetes.windows.md]: Kubernetes _Windows_ Walkthrough?

File: examples/README.md [hunk 3]
```diff
@@ -16,4 +16,4 @@ This cluster definition examples demonstrate how to create a customized Docker E
 * [Attached Disks](disks-storageaccount) - shows how to attach up to 4 disks per node
 * [Managed Disks](disks-managed) (under private preview) - shows how to use managed disks 
 * [Large Clusters](largeclusters) - shows how to create cluster sizes of up to 1200 nodes
-* [Windows Clusters](windows) - shows how to create windows or mixed Microsoft Windows and Linux Docker clusters on Microsoft Azure
```
Comment (JiangtianLi) [inline examples/README.md]: upper case **W**indows

File: docs/kubernetes-build-win-binaries.md (hunk 1 above)
Comment (anhowe) [inline docs/kubernetes-build-win-binaries.md]: fixed.

File: docs/kubernetes.windows.md (hunk 2 above)
Comment (anhowe) [inline docs/kubernetes.windows.md]: fixed.

File: examples/README.md (hunk 3 above)
Comment (anhowe) [inline examples/README.md]: I've removed "windows or"

**Comment by anhowe** [comment]
@rjtsdl I've verified by deploying many clusters, and going through the walkthrough from scratch.

**Pull request merged**